strawberryfields/io.py

```diff
@@ -83,7 +83,12 @@ def to_blackbird(prog, version="1.0"):
             for a in cmd.op.p:
                 if sfpar.par_is_symbolic(a):
                     # SymPy object, convert to string
-                    a = str(a)
+                    isMeasuredParameter = lambda x : isinstance(x, sfpar.MeasuredParameter)
```

Comment:
One more comment: Should we define this lambda function outside the for loop, so we don't have to redefine it each time?